source/assets/img/arrow-up.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="21px" height="10px" viewBox="0 0 21 10" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 48.2 (47327) - http://www.bohemiancoding.com/sketch -->
```

Comment:
Please get rid of the meta information. You can use a SVG minifier like http://www.svgminify.com/ or a GUI tool like https://imageoptim.com/mac

Reply:
check